gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -1332,6 +1331,12 @@ gum_exec_ctx_load_real_register_into (GumExecCtx * ctx,
   gint slot_in_the_stack;
   gint pos_in_the_slot;
 
+#if STALKER_DEBUG_LEVEL == 10
+  g_print("gum_exec_ctx_load_real_register_into - start\n");
+  g_print("\nsource: %d\n", source_register);
+  g_print("\ntarget: %d\n", target_register);
```

Comment:
Missing space between function name and opening parenthesis on these three lines.